### PR TITLE
Bench 216: Checkout Page

### PR DESCRIPTION
**Pull request by harrystead2**
burgerdemo2

### CODE REVIEW

File: src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -29,6 +29,12 @@ export const Navigation = (): ReactElement => {
         >
           Staff
         </NavLink>
+        <NavLink
+          className={({ isActive }) => cn('nav__item', { 'nav__item--active': isActive })}
+          to={RouteConstants.CHECKOUT}
+        >
+          Checkout
+        </NavLink>
```
Comment (JavedK15) [inline src/components/Navigation/Navigation.tsx]: Do we want this in the nav bar? I feel like it should live on the order summary on the right of the screen?

**Comment by JavedK15** [comment]
There's a few build warnings we can get rid of

File: src/components/Navigation/Navigation.tsx (hunk 1 above)
Comment (JavedK15) [reply src/components/Navigation/Navigation.tsx]: I see that it already does. I just don't think we need it on the nav bar as well

**Review by mbaltrimas — DISMISSED** [review]

**Comment by JavedK15** [comment]
Are we still getting build warnings? We should be looking to remove those to keep the code clean. It seems to be mainly for unused variables. Make sure to check the build log before pushing for PR, will speed things up a bit 👍

**Review by JavedK15 — DISMISSED** [review]

File: src/components/OrderCreateForm/OrderCreateForm.tsx [hunk 2]
```diff
@@ -2,20 +2,27 @@ import './OrderCreateForm.scss';
 import { Button } from 'components/Button/Button';
 import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
 import { useOrder } from 'hooks/useOrder';
-import { FormEvent, ReactElement } from 'react';
+import { FormEvent, ReactElement, useState } from 'react';
```
Comment (mbaltrimas) [inline src/components/OrderCreateForm/OrderCreateForm.tsx]: one more warning

**Review by mbaltrimas — CHANGES_REQUESTED** [review]

**Review by mbaltrimas — APPROVED** [review]

**Review by JavedK15 — APPROVED** [review]

**Pull request merged**